ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -244,6 +245,16 @@ PlatformImeController* WaylandWindow::GetPlatformImeController() {
   return nullptr;
 }
 
+void WaylandWindow::StartWindowMoveOrResize(int hittest,
+                                            gfx::Point pointer_location) {
+  connection_->ResetPointerFlags();
```

Comment:
WDYT about adding 'DCHECK(xdg_surface_);' here as other APIs to use xdg_surface_ have it?

Reply:
done